tf-modules/ingest/message_template.tf

```diff
@@ -43,11 +43,12 @@ resource "aws_iam_role_policy" "lambda_processing_role_get_secrets" {
 
 locals {
   default_queues = {
-    triggerLambdaFailure      = aws_sqs_queue.trigger_lambda_failure.id
-    startSF                   = aws_sqs_queue.start_sf.id
     backgroundProcessing      = aws_sqs_queue.background_processing.id
     kinesisFailure            = aws_sqs_queue.kinesis_failure.id
+    reporting                 = var.cw_sf_event_to_db_records_sqs_queue_url
```

Comment:
I'm probably forgetting something, but is there a value/necessity in including the queue URL in the list of queues in the message?

Reply:
If user tasks wanted to use it, they could, I suppose.

Reply:
I don't have a real strong justification other than it seemed like the same kind of thing we already expose in this block.

Reply:
Yeah, I can't remember if there's a reason why queues need to be here in the message, except in the case of throttle queues which are looked up from the message.

The only concern I guess would be if this queue URL is in the Cumulus message then tasks might publish to it directly rather than using the `sf-sqs-report` task. Is that a bad thing?

Reply:
Not as long as they follow the stipulations laid out in `ingest-notifications.md`

Reply:
Not as long as they follow the stipulations laid out in `ingest-notifications.md`